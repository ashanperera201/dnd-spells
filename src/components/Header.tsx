import React from 'react';
import Typography from '@mui/material/Typography';
import './styles/Header.scss';

const Header = (props: any) => {
    return (
        <Typography className='header-container' variant='h3' margin={2} >
            DND 5 Spells
        </Typography>
    )
}

export default Header;