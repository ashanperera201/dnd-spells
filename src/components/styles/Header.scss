@import "../../styles/responsive";

.header-container {
    text-align: start;

    @include respond(phone) {
        text-align: center !important;
    }
}
