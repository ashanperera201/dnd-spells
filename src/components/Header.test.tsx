import renderer from 'react-test-renderer';
import { render } from '@testing-library/react';
import Header from './Header';

it('Should be render header component properly.', () => {
    const tree = renderer.create(
        <Header />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});

it('Should contain the header', () => {
    const { getAllByText } = render(<Header />);
    expect(getAllByText("DND 5 Spells")).toBeTruthy();
})





