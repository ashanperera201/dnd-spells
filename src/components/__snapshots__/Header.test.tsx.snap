// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should be render header component properly. 1`] = `
<h3
  className="MuiTypography-root MuiTypography-h3 header-container css-1mocu2h-MuiTypography-root"
>
  DND 5 Spells
</h3>
`;
